packages/@tailwindcss-upgrade/src/index.ts

```diff
@@ -140,9 +142,10 @@ async function run() {
     // Migrate source files, linked to config files
     {
       // Template migrations
-
-      info('Migrating templates using the provided configuration file.')
       for (let config of configBySheet.values()) {
+        info(
+          `Migrating templates using the provided configuration file: ${highlight(relative(config.configFilePath, base))}.`,
+        )
```

Comment:
We could be dealing with template files originating from different Tailwind config files, therefore this log has been moved inside of the loop.

Slightly updated the message and included the linked config file.